### ISSUE TITLE
Do not show versions with specific keywords

### ISSUE THREAD

**Issue by xahem** [ImranR98/Obtainium#1416]
I would like to track snapchat updates with obtanium, but the problem is that every source has both stable and beta releases. I would like to be able to have the option to prevent obtanium from telling me a new beta version is released by being able to add the option not to search for the keyword "Beta", or in case of X for example, not to search for the keyword "Alpha"
![Screenshot_20240221-114323](https://github.com/ImranR98/Obtainium/assets/97387346/876f7c45-4c99-471b-b60f-e890070fe580)

**Comment by xahem** [comment]
> Hey @xahem ,
> Is the option “Include prereleases” in the “Additional Settings” of the respective app not working sufficiently? So Obtainium still recognizes beta releases even though the toggle is deactivated? Or were you until now not aware that there is a toggle?

Hello, I don't have that option. See attached screenshot.![Screenshot_20240222-123451.png](https://github.com/ImranR98/Obtainium/assets/97387346/df402fb3-5f08-4425-8457-29385a8849a8)


![Screenshot_20240222-123323.png](https://github.com/ImranR98/Obtainium/assets/97387346/f09b48ce-e96e-46d1-a6a6-bfa2e6fc3da0)

**Comment by DwainZwerg** [comment]
Good point: The function is not available for apkmirror.com, apkpure.net and uptodown.com; presumably this also applies to aptoide.com (which I don’t use)
Sorry, I wasn't aware of this until now because I always update all apps to the latest beta version.
Let’s see what ImranR98 says.

**Comment by xahem** [comment]
> Good point: The function is not available for apkmirror.com, apkpure.net and uptodown.com; presumably this also applies to aptoide.com (which I don’t use) Sorry, I wasn't aware of this until now because I always update all apps to the latest beta version. Let’s see what ImranR98 says.

It's not a problem. it's because I patch some apps, and some patches only work with stable versions, so I would like to be able to track new stable releases via obtanium as now the only way I find out a stable release is out is by opening APKmirror every couple of days
